### PR TITLE
Plotter for the PELEPlatform

### PR DESCRIPTION
**Pull request by martimunicoy**

### CODE REVIEW

**Comment by martimunicoy** [comment]
work in progress, not finished yet

File: pele_platform/plotter.py [hunk 1]
```diff
@@ -0,0 +1,1374 @@
...
+                        help='Highest value to plot on the X axis')
+    parser.add_argument('--ylowest', type=float, default=None,
+                        help='Lowest value to plot on the Y axis')
+    parser.add_argument('--yhighest', type=float, default=None,
+                        help='Highest value to plot on the Y axis')
+    parser.add_argument('--zlowest', type=float, default=None,
+                        help='Lowest value to plot on the Z axis')
+    parser.add_argument('--zhighest', type=float, default=None,
+                        help='Highest value to plot on the Z axis')
+    parser.add_argument('--colormap', type=str, default='gnuplot',
```
Comment (frumpowy) [inline pele_platform/plotter.py:517]: Add a list of available colour maps to command line help.

**Comment by frumpowy** [comment]
Can we make all plots with the same theme? I feel like there was an easy way to use seaborn style in matplotlib. The density one is so beautiful, would be nice to have the other ones like that as well.

File: pele_platform/plotter.py (hunk 1 above)
Comment (martimunicoy) [reply pele_platform/plotter.py:530]: It is going to be already in the documentation and the list of arguments is already too large. For now I would not add them there.

**Pull request merged**